unittests/protocol_feature_tests.cpp

```diff
@@ -2432,4 +2432,101 @@ BOOST_AUTO_TEST_CASE(sync_call_activation_test) try {
                        c.success());
 } FC_LOG_AND_RETHROW()
 
+BOOST_AUTO_TEST_CASE_TEMPLATE(packed_transaction_restrictions_test, T, testers) { try {
```

Comment:
There is no point in using `testers` here when the tester is initialized with `setup_policy::preactivate_feature_and_new_bios` or `setup_policy::none`, since `legacy_tester` and `savanna_tester` are different only when initialized with `setup_policy::full`.

I'd suggest just using `tester` in a `BOOST_AUTO_TEST_CASE`
I'd suggest just using tester in a `